packages/orchestration/src/withdraw-reward.js

```diff
@@ -0,0 +1,72 @@
+// @ts-check
+
+// question: are we doing snake-case filenames?
```

Comment:
It seems we've adopted camelCase, outside of `start-` `init-` and `-proposal` files

Reply:
I retract this statement, and will start avoiding `camelCase` in filenames keeping in mind case insensitive file systems